masterlist.yaml

```
@@ -12034,6 +12038,15 @@ plugins:
         util: '[SSEEdit v4.0.4](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 23
 
+  - name: 'Pilgrim.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/54099' ]
```

Links should have trailing slashes where possible for stylistic consistency (except in cleaning data). They'd benefit from names when displayed in the UI as well:
```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/54099/'
        name: 'Pilgrim - A Religion Overhaul'
```